docker/mysql/Dockerfile

```diff
@@ -1,5 +1,37 @@
 FROM mysql
```

Comment:
Just a thought:

May be we should pick a version (to make this `Dockerfile` more stable through time).